docs/resources/Guild.md

```diff
@@ -276,6 +276,54 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
 }
 ```
 
+### Get Guild Widget Object
+
+###### Get Guild Widget Structure
+
+| Field                      | Type                                                                                | Description                                                 |
+| -------------------------- | ----------------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| id                 | snowflake                                                                         | guild id                                                             |
+| name               | string                                                                            | guild name (2-100 characters)                                        |
+| instant_invite     | ?string                                                                           | instant invite for the guilds specified widget invite channel        |
+| channels           | array of partial [channel](#DOCS_RESOURCES_CHANNEL/channel-object) objects        | voice and stage channels which are accessible by @everyone           |
+| members            | array of partial [user](#DOCS_RESOURCES_USER/user-object) objects                 | special widget user objects that includes users presence (Limit 100) |
+| presence_count     | integer                                                                           | number of online members in this guild                               |
+
+###### Example Get Guild Widget
+
+```json
+{
+  "id": "290926798626999250",
+  "name": "Test Server",
+  "instant_invite": "https://discord.com/invite/abcdefg",
+  "channels": [
+    {
+      "id": "705216630279993882",
+      "name": "elephant",
+      "position": 2
+    },
+    {
+      "id": "669583461748992190",
+      "name": "groovy-music",
+      "position": 1
+    }
+  ],
+  "members": [
+    {
+      "id": "0",
+      "username": "1234",
+      "discriminator": "0000",
+      "avatar": null,
+      "status": "online",
+      "avatar_url": "https://cdn.discordapp.com/widget-avatars/FfvURgcr3Za92K3JtoCppqnYMppMDc5B-Rll74YrGCU/C-1DyBZPQ6t5q2RuATFuMFgq0_uEMZVzd_6LbGN_uJKvZflobA9diAlTjhf6CAESLLeTuu4dLuHFWOb_PNLteooNfhC4C6k5QgAGuxEOP12tVVVCvX6t64k14PMXZrGTDq8pWZhukP40Wg"
+    }
+  ],
+  "presence_count": 1
+}
+```
+> warn
+> The fields `id`, `discriminator` and `avatar` are anonymized to prevent abuse.
```

Comment:
I think this warning should be below the model definition instead of below the example

Reply:
Done 👍